pkgs/applications/editors/emacs/build-support/melpa.nix

```diff
@@ -126,6 +126,13 @@ EOF
   buildPhase = args.buildPhase or ''
     runHook preBuild
 
+    # This is copied from stdenv buildPhase. foundMakefile is used in stdenv checkPhase.
+    if [[ -z "''${makeFlags-}" && -z "''${makefile:-}" && ! ( -e Makefile || -e makefile || -e GNUmakefile ) ]]; then
+      echo "no Makefile"
```

Comment:
```suggestion
      nixInfoLog "no Makefile found"
```

Reply:
Actually, I think this message is not very helper in our buildPhase.  What about deleting it?

Reply:
Yes, I would prefer to remove it.